### PR TITLE
Fix: Book bundle showing wrong amount of books

### PR DESCRIPTION
**Pull request by ItsEmpa**
## Dependency:
- https://github.com/hannibal002/SkyHanni-REPO/pull/120

https://ptb.discord.com/channels/997079228510117908/1210198335408046130

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueCalculator.kt [hunk 1]
```diff
@@ -576,7 +585,7 @@ object EstimatedItemValueCalculator {
                 level = 1
             }
             if (internalName.startsWith("ENCHANTED_BOOK_BUNDLE_")) {
-                multiplier = 5
+                multiplier = if(bookBundleAmount.contains(rawName)) bookBundleAmount[rawName]!! else 5
```
Comment (jani270) [inline src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueCalculator.kt]: That looks like a formatting moment at the start

**Comment by jani270** [comment]
Also what the fuck ptb discord

**Comment by ItsEmpa** [comment]
i just have discord, discord ptb and discord canary, ptb i use usually and normal discord/canary i use for my alt sometimes

File: src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueCalculator.kt [hunk 2]
```diff
@@ -540,6 +540,15 @@ object EstimatedItemValueCalculator {
         "ADVANCED_GARDENING_AXE".asInternalName(),
     )
 
+    private val bookBundleAmount = mapOf(
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueCalculator.kt]: move this map into the repo (i know it is annoying for so small and not important stuff, but it still helps consistency and maintainability in a big way)

**Comment by hannibal002** [comment]
I don't know when you have actually started creating this pr., but please try to always update your beta branch before creating the new branch.
right now, those are the commits missing in your branch; 
grafik

(this is no problem for this case, but there could be merge conflicts that happened during those last few commits. easily preventable)

File: src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueCalculator.kt [hunk 3]
```diff
@@ -576,7 +585,7 @@ object EstimatedItemValueCalculator {
                 level = 1
             }
             if (internalName.startsWith("ENCHANTED_BOOK_BUNDLE_")) {
-                multiplier = 5
+                multiplier = if (bookBundleAmount.contains(rawName)) bookBundleAmount[rawName]!! else 5
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueCalculator.kt]: use `bookBundleAmount.getOrDefault(rawName, 5)` instead

**Comment by ItsEmpa** [comment]
ah, alr. yeah i started doing it a couple days ago but couldnt log on hypixel to test it

File: src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueCalculator.kt (hunk 3 above)
Comment (Thunderblade73) [inline src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueCalculator.kt:588]:
```suggestion
                multiplier = bookBundleAmount[rawName] ?: 5
```

Comment (hannibal002) [reply src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueCalculator.kt]: thunder was faster

**Review by hannibal002 — CHANGES_REQUESTED** [review]
works as good in game.

File: src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueCalculator.kt (hunk 3 above)
Comment (Thunderblade73) [reply src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueCalculator.kt]: but hannibal was better

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**